### PR TITLE
Added support for `add_respondents_data_without_email_invitation`

### PR DESCRIPTION
**Pull request by matsve**
We needed this function, and thought that it would be nice to contribute back to your helpful gem. Thanks :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Simple Ruby client for QuestBack's SOAP API.
 
-[![Build Status](https://travis-ci.org/Skalar/quest_back.svg?branch=master)](https://travis-ci.org/Skalar/quest_back)
+[ ![Codeship Status for matsve/quest_back](https://codeship.com/projects/74070280-df99-0133-54e5-460ef9277ccd/status?branch=master)](https://codeship.com/projects/145033)
```
Comment (thhermansen) [inline README.md]: Please revert the README changes so the build status remains travis :-)

**Comment by thhermansen** [comment]
Hi, and thank you for the pull request.

Here are a few comments/requests, in addition to the comment on the `README` file.

* Please merge Skalar's master in to your branch, as I have fixed travis configuration. This way we'll get a working CI status on your pull request.
* Please add a test to the method you added. You can take a look at how we did it for the [add_respondents_data here](https://github.com/Skalar/quest_back/blob/master/spec/quest_back/api_spec.rb#L172-L255).

**Comment by thhermansen** [comment]
I'm closing this for now as I didn't get any reply from @matsve. Feel free to reopen a pull request when my [comments](https://github.com/Skalar/quest_back/pull/2#issuecomment-207385073) are resolved.

**Comment by matsve** [comment]
Hi, @thhermansen, I apologise in the delay regarding this. I have now made the tests and fixed the readme to use travis again. It's in pull request #3

**Pull request closed without merging**